testdata/test_mg_ac_normalized.txt

```diff
@@ -0,0 +1 @@
+amin'ny
```

Comment:
This is great! Even though it's a bit redundant, I think your preprocess test will be easier to read if you just put these input strings directly in the test file rather than reading them from these txt files (it's ok to keep them here as well for the eventual end-to-end test). Obviously the UD one is a bit long so keeping that one in the txt file SGTM.

Reply:
That makes sense. Yeah, the normalized text should be short enough and in an easy enough format to just put it directly in the test file. I think the only input that will need to be in a separate file is actually the UD data.